src/SFA.DAS.EmployerAccounts.Web/Helpers/ControllerConstants.cs

```diff
@@ -2,7 +2,9 @@
 {
     public static class ControllerConstants
     {
-        public const string AboutYourAgreement = "AboutYourAgreement";
+        public const string AboutYourAgreementActionName = "AboutYourAgreement";

```

Comment:
its probably a bigger refactoring exercise, but splitting these into two classes for view names and actions, and not calling the property names xxxViewName, you could possibly do:
```
public static string AboutYourAgreement => nameof(AboutYourAgreement);
```

Reply:
I agree, it's got to the size where we should split it out but not for now.  I'll do it the next time I'm in that class.